caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -59,6 +59,11 @@ func parseBind(h Helper) ([]ConfigValue, error) {
 //         protocols <min> [<max>]
 //         ciphers   <cipher_suites...>
 //         curves    <curves...>
+//         clients {
+//             mode	              [request|require|verify_if_given|require_and_verify]
```

Comment:
Fix the comment alignment here

Reply:
no problem, but which style do you prefer? I currently use something like a.
```
aaa [aaa aaa aaa] {
    aaaaaaa <aaa>
    aaaaa   <aaa...>
    aaaa {
        aaaaaa <aaaaa>
        aaa    <aaaa>
    }
}

bbb [bbb bbb bbb] {
    aaaaaaa <bbb>
    bbbbb   <bbb...>
    bbbb {
            bbbbbb <bbbbb>
            bbb    <bbbb>
    }
}


ccc [ccc ccc ccc] {
    aaaaaaa    <ccc>
    ccccc      <ccc...>
    cccc {
        cccccc <ccccc>
        ccc    <cccc>
    }
}
```